decidim-proposals/spec/system/admin/highlighted_proposal_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Highlighted proposal", type: :system do
+  let(:admin) { create :user, :admin, :confirmed }
+  let(:organization) { admin.organization }
+  let!(:participatory_process) { create(:participatory_process, organization: organization) }
+  let!(:proposal_component) { create(:proposal_component, participatory_space: participatory_process) }
+  let!(:proposals) do
+    [
+      create(:proposal, :not_answered, id: 1, created_at: 9.days.ago, published_at: 9.days.ago, component:
```

Comment:
```suggestion
      create(:proposal, :not_answered, created_at: 9.days.ago, published_at: 9.days.ago, component:
```
better not to specify the id, it can be problematic with the database